variables.tf

```diff
@@ -54,13 +54,25 @@ variable "cognito_iam_role_arn" {
 variable "vpc_id" {
   type        = string
   description = "ID of the VPC where resources will be deployed to"
+  default     = null
 }
 
 variable "subnet_ids" {
   description = "List of Subnet IDs to assign OpenSearch"
   type        = list(string)
+  default     = []
 }
 
+variable "allowed_cidr_blocks" {
+  type        = list(string)
+  default     = []
+  description = "List of CIDR blocks to be allowed to connect to the cluster"
+}
+variable "vpc_enabled" {
+  type        = bool
+  description = "Set to false if ES should be deployed outside of VPC."
+  default     = true
+}
```

Comment:
Can you change the variable name as `enable_public_access` ?

Reply:
if vpc enabled is true then that means ES is in vpc and if its disabled then its outside vpc i.e on internet
so do you think this will make sense?

```
  vpc_id                   = var.enable_public_access ? var.vpc_id : null
  subnet_ids           = var.enable_public_access ? var.subnet_ids : null
  vpc_enabled        = var.enable_public_access

```

Reply:
please check

`  vpc_id                   = var.enable_public_access ? null : var.vpc_id
  subnet_ids           = var.enable_public_access ? null : var.subnet_ids
  vpc_enabled        = var.enable_public_access ? false : true`